This patch makes Logisim Evolution only show the Basys3 board in the board selection dialog.

diff --git a/src/main/java/com/cburch/logisim/fpga/settings/BoardList.java b/src/main/java/com/cburch/logisim/fpga/settings/BoardList.java
index 880496a35..3c032a255 100644
--- a/src/main/java/com/cburch/logisim/fpga/settings/BoardList.java
+++ b/src/main/java/com/cburch/logisim/fpga/settings/BoardList.java
@@ -35,7 +35,7 @@ public class BoardList {
     } else {
       ret.addAll(getBoardsfromJar(pattern, match, file));
     }
-    return ret;
+    return ret.stream().filter(board -> getBoardName(board).equals("BASYS3")).toList();
   }
 
   private static Collection<String> getBoardsfromDirectory(
diff --git a/src/main/java/com/cburch/logisim/prefs/FpgaBoards.java b/src/main/java/com/cburch/logisim/prefs/FpgaBoards.java
index a6b4f512e..abdaeb24f 100644
--- a/src/main/java/com/cburch/logisim/prefs/FpgaBoards.java
+++ b/src/main/java/com/cburch/logisim/prefs/FpgaBoards.java
@@ -391,6 +391,7 @@ public class FpgaBoards implements ActionListener {
     }
     boardSelector.repaint();
     boardSelector.addActionListener(this);
+    boardSelector.setVisible(false);
   }
 
   /**
